recipe/build.sh

```diff
@@ -3,17 +3,27 @@ set -euxo pipefail
 
 rm -rf build || true
 
+
 if [ ${cuda_compiler_version} != "None" ]; then
   CUDA_CMAKE_OPTIONS="-DCMAKE_CUDA_COMPILER=${CUDA_HOME}/bin/nvcc -DCMAKE_CUDA_HOST_COMPILER=${CXX} -DCUDA_ARCH_NAME=All "
   USE_CUDA=ON
+  # Remove -std=c++17 from CXXFLAGS for compatibility with nvcc
+  export CXXFLAGS="$(echo $CXXFLAGS | sed -e 's/ -std=[^ ]*//')"
+  export CFLAGS="$(echo $CFLAGS | sed -e 's/ -mtune=[^ ]*//')"
 else
   CUDA_CMAKE_OPTIONS=""
   USE_CUDA=OFF
+  CFLAGS="${CFLAGS} -std=c17 "
+  CXXFLAGS="${CXXFLAGS} -std=c++17 "
```

Comment:
Would suggest dropping the trailing space

```suggestion
  CFLAGS="${CFLAGS} -std=c17"
  CXXFLAGS="${CXXFLAGS} -std=c++17"
```

Reply:
I am actually not sure if these are needed anymore, if it compiles without them, I will drop them.